Using cpu device
Wrapping the env with a `Monitor` wrapper
Wrapping the env in a DummyVecEnv.
Logging to runs/butlwx68\runs/butlwx68_1







Traceback (most recent call last):
  File "c:\Users\domin\Desktop\Year 2 Block B\2023-24b-fai2-adsai-DominikSzewczyk224180\2. Robotics\Y2B-2023-OT2_Twin-main\task_11_test.py", line 67, in
<module>
    model.learn(total_timesteps=timesteps, callback=wandb_callback, progress_bar=True, tb_log_name=f"runs/{run.id}")
  File "C:\Users\domin\anaconda3\envs\tf\lib\site-packages\stable_baselines3\ppo\ppo.py", line 315, in learn
    return super().learn(
  File "C:\Users\domin\anaconda3\envs\tf\lib\site-packages\stable_baselines3\common\on_policy_algorithm.py", line 277, in learn
    continue_training = self.collect_rollouts(self.env, callback, self.rollout_buffer, n_rollout_steps=self.n_steps)
  File "C:\Users\domin\anaconda3\envs\tf\lib\site-packages\stable_baselines3\common\on_policy_algorithm.py", line 200, in collect_rollouts
    if not callback.on_step():
  File "C:\Users\domin\anaconda3\envs\tf\lib\site-packages\stable_baselines3\common\callbacks.py", line 114, in on_step
    return self._on_step()
  File "C:\Users\domin\anaconda3\envs\tf\lib\site-packages\stable_baselines3\common\callbacks.py", line 219, in _on_step
    continue_training = callback.on_step() and continue_training
  File "C:\Users\domin\anaconda3\envs\tf\lib\site-packages\stable_baselines3\common\callbacks.py", line 114, in on_step
    return self._on_step()
  File "C:\Users\domin\anaconda3\envs\tf\lib\site-packages\wandb\integration\sb3\sb3.py", line 142, in _on_step
    self.save_model()
  File "C:\Users\domin\anaconda3\envs\tf\lib\site-packages\wandb\integration\sb3\sb3.py", line 151, in save_model
    wandb.save(self.path, base_path=self.model_save_path)
  File "C:\Users\domin\anaconda3\envs\tf\lib\site-packages\wandb\sdk\wandb_run.py", line 371, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\domin\anaconda3\envs\tf\lib\site-packages\wandb\sdk\wandb_run.py", line 361, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\domin\anaconda3\envs\tf\lib\site-packages\wandb\sdk\wandb_run.py", line 1852, in save
    return self._save(glob_str, base_path, policy)
  File "C:\Users\domin\anaconda3\envs\tf\lib\site-packages\wandb\sdk\wandb_run.py", line 1906, in _save
    os.symlink(abs_path, wandb_path)
OSError: [WinError 1314] A required privilege is not held by the client: 'c:\\Users\\domin\\Desktop\\Year 2 Block
B\\2023-24b-fai2-adsai-DominikSzewczyk224180\\2. Robotics\\Y2B-2023-OT2_Twin-main\\models\\butlwx68\\model.zip' -> 'c:\\Users\\domin\\Desktop\\Year 2 Block
B\\2023-24b-fai2-adsai-DominikSzewczyk224180\\2. Robotics\\Y2B-2023-OT2_Twin-main\\wandb\\run-20240119_191628-butlwx68\\files\\model.zip'
[35m  10%[39m [38m━━━━━━━━━━╸━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[39m [32m998/10,000 [39m [ [33m0:00:16[39m < [36m0:02:32[39m , [31m59 it/s[39m ]